CONTRIBUTING.md

```diff
@@ -27,6 +27,7 @@ The first step is to develop and test your changes locally.
 ##### 1. Build Kontrol from source
 ```
 kup install k.openssl.procps --version v$(cat deps/k_release)
+kup install k.openssl.procps.secp256k1 --version v$(cat deps/k_release)
```

Comment:
I think we could just replace the initial command.

```suggestion
kup install k.openssl.procps.secp256k1 --version v$(cat deps/k_release)
```
Can you update the instructions in the README.md as well?